web/gds-user-ui/src/modules/dashboard/organization/SwitchOrganization.tsx

```diff
@@ -29,12 +29,11 @@ const SwitchOrganization: React.FC = () => {
           orgid: id
         });
         if (logged.status === APP_STATUS_CODE.NO_CONTENT) {
-          const PromiseArray = [refreshNewToken(), getUserCurrentOrganizationService()];
-          const [, user] = await Promise.all(PromiseArray);
+          const token = (await refreshNewToken()) as any;
+          const user = token && (await getUserCurrentOrganizationService());
```

Comment:
Is this a change to the logic here? I'm not too familiar with the code so I can't tell what it's meant to be doing.

Reply:
no, it's to figure out what is going on in staging. i will remove it later.